sdk/python/feast/feature_service.py

```diff
@@ -14,16 +16,31 @@
 
 
 class FeatureService:
+    """
+    A FeatureService object is used to group features together for retrieval.
```

Comment:
Nitpick, but we know it's a `FeatureService` object by looking at the class name. In general I prefer us to talk about what this class does instead of adding the name of the object/class into the description.

`A feature service allows users to group features for offline or online retrieval`